### ISSUE TITLE
Throws with Storyshots / Jest because of Babel Helpers being ESM

### ISSUE THREAD

**Issue by swernerx** [tuchk4/storybook-readme#147]
In our current setup Storybooks v5 works fine, but Storyshots with Jest broke. Jest requires that all imports are CommonJS. This is doable for our local code with a Jest transformer, but not for 3rd party code from `node_modules`. 

Current output:

```
$ jest --config jest.storyshots.config.js
 FAIL  src/index.storyshots.tsx
  ● Test suite failed to run

    /Users/swerner/Workspace/witt/emma-components/node_modules/@babel/runtime/helpers/esm/toConsumableArray.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import arrayWithoutHoles from "./arrayWithoutHoles";
                                                                                                    ^^^^^^^^^^^^^^^^^

    SyntaxError: Unexpected identifier

      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:471:17)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:513:25)
      at Object.<anonymous> (node_modules/storybook-readme/services/getDocsLayout.js:10:50)
```

Jest Config: `jest.storyshots.config.js`

```
module.exports = {
  testMatch: ['**/?(*.)storyshots.tsx'],
  transform: {
    '^.+\\.tsx?$': 'babel-jest',
    '^.+\\.md?$': 'markdown-loader-jest',
  },
}
```

I see that the `.babelrc` is based upon `react-app` babel preset. This might be fine in Webpack environments. But as Jest Storyshots is happening outside of Webpack this breaks.

Looks like the fix is pretty trivial by disabling the ESM mode for `@babel/plugin-transform-runtime`. Luckily `react-app` offers a setting for this:

By applying this change everything works as desired in Jest:

```diff
diff --git a/packages/storybook-readme/.babelrc b/packages/storybook-readme/.babelrc
index 2cb2bc4..2d11a51 100644
--- a/packages/storybook-readme/.babelrc
+++ b/packages/storybook-readme/.babelrc
@@ -1,4 +1,4 @@
 {
-  "presets": [["react-app", { "absoluteRuntime": false }]],
+  "presets": [["react-app", { "absoluteRuntime": false, "useESModules": false }]],
   "plugins": ["@babel/plugin-transform-modules-commonjs"]
 }
```

**Comment by swernerx** [comment]
I extracted this for easy apply into a PR: https://github.com/tuchk4/storybook-readme/pull/148

**Comment by tuchk4** [comment]
Thanks!

Will publish new version with this fix at the end of this week

**Comment by tuchk4** [comment]
published 5.0.2
